pkg/plugins/policies/meshmetric/plugin/v1alpha1/plugin.go

```diff
@@ -268,3 +283,7 @@ func filterPrometheusBackends(backends *[]api.Backend) []*api.PrometheusBackend
 	}
 	return prometheusBackends
 }
+
+func backendNameFrom(endpoint string) string {
```

Comment:
I am not sure about this backend name on Kubernetes it looks like: `otel-collector.observability.svc4317` it is unique for backend, and I didn't find any issues with it yet. But this ends up as part of listener and cluster name. Can this end up too long, or be problematic because of characters like: `.` or `-`?

Reply:
From docs:

> A string must always contain UTF-8 encoded or 7-bit ASCII text, and cannot be longer than 2^32.

So I don't think it's ever going to be longer than `4 294 967 296`